03-task-manager/starter/controllers/people.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const router = express.Router();
+
+require('dotenv').config();
+
+
+const Person = require('../models/Person');  // Adjust path if necessary
```

Comment:
Here you were trying to get the "Person" model, but there is only "Task.js" in the models folder.